<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenEffectFileClass</class>
 <widget class="QMainWindow" name="GenEffectFileClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenEffectFile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>481</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>批量打包</string>
    </property>
    <widget class="QPushButton" name="openfileBtn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件夹</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBtn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存路径</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startBtn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始转换</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="openDirlineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="saveDirLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="versionedit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="versionBtn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>版本文件路径</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>557</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="geneffectfile.qrc"/>
 </resources>
 <connections/>
</ui>
